.npmignore

```diff
@@ -37,3 +37,6 @@ android/gradle/
 android/gradlew
 android/gradlew.bat
 android/local.properties
+
+# Vendor iOS Mapbox.framework
+!ios/Mapbox.framework
```

Comment:
This includes the framework in the tarball that will be sent to the npm registry